Ginger/GingerCoreNET/Run/Context.cs

```diff
@@ -112,6 +131,144 @@ public TargetBase Target
             }
         }
 
+        private string mTargetApplication;
+        public string TargetApplication
+        {
+            get
+            {
```

Comment:
no need TargetApplication we already have "Target" we should represent the target application in context

Reply:
Done